Source/items.cpp

```diff
@@ -2795,7 +2785,7 @@ void items_427ABA(Point position)
 {
 	PkItemStruct PkSItem;
 
-	if (CornerStone.activated || position.x == 0 || position.y == 0) {
+	if (CornerStone.activated || position == Point{0, 0}) {

```

Comment:
This effectively change it from or to and, is this is intended?

Reply:
It was a mistake. Fixed.